wiki_articles/cp.md

```diff
@@ -1,38 +1,28 @@
-<!-- alias CP-help, CP -->
-
+<!-- alias cp-help, cp -->
 # The difficulty in helping with competitive coding problems
```

Comment:
The project uses Wikipedia title case, see https://titlecaseconverter.com/ for help.